packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+### List members in a binding
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}/members" />
+
+Get a paginated list of Members assigned to a Role in a Resource Set
+
+#### Request parameters
+
+| Parameter      | Description                                                           | Param Type   | DataType       | Required |
+| :------------- | :-------------------------------------------------------------------- | :----------- | :------------- | :------- |
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | the `id` of the Role to identify the Binding                          | URL          | String         | TRUE     |
+
+#### Response parameters
+
+A paginated list of [Members](#member-object).
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings/cr0WxyzJxGIr0ouum0g4/members"
+```
+
+#### Response example
+
+```json
+{
+  "members": [
+    {
+      "id": "irb1qe6PGuMc7Oh8N0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/users/00uuk41Hjga5qGfQ30g3"
+        }
+      }
+    }, {
+      "id": "irb1q92TFAHzySt3x0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "binding": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/${resource-set-id}/bindings/${role-id-or-name}"
+    },
+    "next": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/${resource-set-id}/bindings/${role-id-or-name}/members?after=${last-member-id}"
+    }
+  }
+}
+```
+### Get a member from a binding
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}/members/${memberId}" />
+
+Get a Member of a Role in a Resource Set
+
+#### Request parameters
+
+| Parameter      | Description                                                           | Param Type   | DataType       | Required |
+| :------------- | :-------------------------------------------------------------------- | :----------- | :------------- | :------- |
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | `id` of the Role to identify the Binding                              | URL          | String         | TRUE     |
+| memberId       | `id` of the Member within the Binding                                 | URL          | String         | TRUE     |
```

Comment:
General comment: When ID is mentioned in the Description, maybe just identify it as ID, not `id`.